staff/Melinx/css-adaptive/css/desktop.css

```diff
@@ -0,0 +1,37 @@
+
+html, body {
```

Comment:
TIP move common styles (mobile / desktop) to main.css

Reply:
done, thx!